WordPress/WordPressTest/FollowCommentsServiceTests.swift

```diff
@@ -0,0 +1,97 @@
+import XCTest
+import WordPressKit
+
+@testable import WordPress
+
+class FollowCommentsServiceTests: XCTestCase {
+
+    private var contextManager: TestContextManager!
+
+    // MARK: - Lifecycle
+
+    override func setUp() {
+        super.setUp()
+        contextManager = TestContextManager()
+    }
+
+    override func tearDown() {
+        ContextManager.overrideSharedInstance(nil)
+        super.tearDown()
+    }
+
+    // MARK: - Mocks / Tester
+
+    class ReaderPostServiceRemoteMock: ReaderPostServiceRemote {
```

Comment:
I often prefer to add the mocks at the end of the file. Wdyt?

Mainly because when I open a test I'm interested in the tested behaviors, here I have to scroll down and find the tests.